src/main/java/org/learntotravel/master/direction/Direction.java

```diff
@@ -0,0 +1,74 @@
+package org.learntotravel.master.direction;
+
+public class Direction {
+
+    private long id;
+
+    private String[] emails;
+
+    private int threshold;
+
+    private String source;
+
+    private SourceType type;
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String[] getEmails() {
+        return emails;
+    }
+
+    public void setEmails(String[] emails) {
+        this.emails = emails;
+    }
+
+    public int getThreshold() {
+        return threshold;
+    }
+
+    public void setThreshold(int threshold) {
+        this.threshold = threshold;
+    }
+
+    public String getSource() {
+        return source;
+    }
+
+    public void setSource(String source) {
+        this.source = source;
+    }
+
+    public SourceType getType() {
+        return type;
+    }
+
+    public void setType(SourceType type) {
+        this.type = type;
+    }
+
+    enum SourceType {
```

Comment:
Maybe we could make it a separate class?

Reply:
Moved to the separate file.